subscription-engine/kafka-ws/app/App.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+
+const expressWS = require('express-ws');
+
+const addRequestId = require('express-request-id');
+
+const compression = require('compression');
+
+const createError = require('http-errors');
+
+const morgan = require('morgan');
+
+/* This library is about what happens when you hit an async error. */
+require('express-async-errors');
+
+const { Logger } = require('@dojot/microservice-sdk');
+
+const { app: appCfg, nodeEnv } = require('./Config');
+
+const app = express();
+const accessControlRouter = require('./routes/AccessControlRouter');
+const ticketRouter = require('./routes/TicketRouter');
+const topicsRouter = require('./routes/TopicsRouter');
+
+function configure(server) {
+  const logger = new Logger();
+  logger.info('Configuring the Express Application...');
+
+  /* Lets it define WebSocket endpoints like any other type of route
+   * and applies regular Express middleware. */
+  expressWS(app, server);
+
+  /* Generate UUID for request and add it to X-Request-Id header.
```

Comment:
Nice, good idea!

It would be interesting to define that the request was part of a context object, thus, each new request would create a context object that could be accessed without having to pass it by parameter everywhere, but I think it would be necessary a framework for that, something more sophisticated than the Express.